endpoints/cinema/index.js

```diff
@@ -2,6 +2,12 @@ const request = require('request')
 const cheerio = require('cheerio')
 const app = require('../../server')
 
+// Utility function
+const scrapeImage = (src) => {
+  const urls = src.match(/\/images\/poster\/.+\.(jpg|jpeg|png)/ig) || ['']
```

Comment:
Should `return null` if `urls` is `null`, for consistency with the check for `movieImage` below. Otherwise it will return `'http://kvikmyndir.is'` if `src` is invalid.

Reply:
That make sense, I'll fix it